extract-l1b.sh

```diff
@@ -30,6 +32,8 @@ fi
 DATE=$1
 YEAR=${DATE:0:4}
 MONTH=${DATE:4:2}
+DAY=${DATE:6:2}
```

Comment:
seems like the files have been changed in the server... thanks for catching that